client/src/components/Auth/PrivateRoute/__tests__/PrivateRoute.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import { Redirect } from "react-router-dom";
+import { mount } from "enzyme";
+import { findByTestAttr } from "../../../../test/testUtils";
+import { GenericComponent } from "../../../../test/__mocks__/fileMock";
+import PrivateRoute from "../PrivateRoute";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore();
+
+const loggedInState = {
+  auth: {
+    isAuthenticated: true
+  }
+};
+
+const loggedOutState = {
+  auth: {
+    isAuthenticated: false
+  }
+};
+
+describe("PrivateRoute rendering", () => {
+  test("PrivateRoute should render a component if user is logged in", () => {
+    const store = mockStore(loggedInState);
+    let wrapper = mount(
+      <MemoryRouter initialEntries={["/dashboard"]} initialIndex={0}>
+        <PrivateRoute
+          path="/dashboard"
+          component={GenericComponent}
+          store={store}
+        />
+      </MemoryRouter>
+    );
+    const generic = findByTestAttr(wrapper, "generic");
+
+    expect(generic.exists()).toBe(true);
+  });
+
+  // TODO - this test is likely not asserting what it should
+  test("PrivateRoute should redicted to the login page if user is logged out", () => {
+    const store = mockStore(loggedOutState);
+    let wrapper = mount(
+      <MemoryRouter initialEntries={["/dashboard"]} initialIndex={0}>
+        <PrivateRoute
+          path="/dashboard"
+          component={GenericComponent}
+          store={store}
+        />
+      </MemoryRouter>
+    );
+    expect(wrapper.find(Redirect)).toEqual({});
```

Comment:
Why does it equal `{}` in this case? Maybe it would be better to mock `Redirect` so you can use `toBe` instead?